src/java.base/share/classes/sun/net/www/ParseUtil.java

```diff
@@ -197,11 +197,14 @@ public static String decode(String s) {
             bb.clear();
             int ui = i;
             for (;;) {
-                assert (n - i >= 2);
+                if (n - i >= 2) {
```

Comment:
OK - the test should be inverted - sorry for not noticing that earlier. It should be `n - i < 2`. It's causing massive failures in the CI. Can you please run at least the tests under jdk/sun/net and jdk/java/net after making the change?

Reply:
I am sorry for my bad mistake.
I fixed it and run the tests under jdk/sun/net and jdk/java/net. I confirmed all tests passed in Linux.